products/compute/terraform.yaml

```diff
@@ -1422,6 +1422,35 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       constants: 'templates/terraform/constants/router_nat.go.erb'
       resource_definition: 'templates/terraform/resource_definition/router_nat.go.erb'
+  RouterBgpPeer: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: 'google_compute_router_peer'
+    exclude_validator: true
+    id_format: "projects/{{project}}/regions/{{region}}/routers/{{router}}/{{name}}"
+    mutex: router/{{region}}/{{router}}
+    examples:
+      # These examples are not used to autogenerate tests, as fine-grained
```

Comment:
Could you also rename the existing file so it matches the name of the generated one and its docs?

Reply:
Not sure how to do that: trying to bring back the files I've already deleted gives me the following error:
```
/Users/drebes/Development/magic-modules/provider/core.rb:142:in `block (2 levels) in copy_file_list': /Users/drebes/Development/go/src/github.com/terraform-providers/terraform-provider-google/website/docs/r/compute_router_bgp_peer.html.markdown was already modified during this run. 2019-11-19 23:37:24 +0000 (RuntimeError)
bundler: failed to load command: compiler (compiler)
```

Reply:
Ah yeah I don't think you need to un-delete anything,  just rename https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/tests/resource_compute_router_peer_test.go

Reply:
You mean rename `resource_compute_router_peer_test.go` to `resource_compute_router_bgp_peer_test.go`?

Reply:
Yup! Sorry, could have been clearer. My intent basically is that once this is merged, anything in the generated codebases relevant to this resource has a consistent name.

Reply:
Thanks. I've pushed my changes.